### PR TITLE
Lesson9

### PR DESCRIPTION
**Pull request by antonbl32**
Robot,Date&Time

### CODE REVIEW

File: src/main/java/by/antonsh/lesson9/Ex28/ObjectEx.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.Scanner;
+/*
+Написать класс, который умеет хранить в себе массив любых типов данных.
+Реализовать метод get(int index), который возвращает любой элемент массива по индексу.
+ */
+
+/**
+ * This class create Object array and fill it.
+ */
+public class ObjectEx {
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson9/Ex28/ObjectEx.java]: we don't need this class, need only Generic

File: src/main/java/by/antonsh/lesson9/robot/Robot.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     * @return int
+     */
+    @Override
+    public int getPrice() {
+        int price = head.getPrice() + hand.getPrice() + leg.getPrice();
+        return price;
+    }
+
+
+
```
Comment (svetamolchanova) [inline src/main/java/by/antonsh/lesson9/robot/Robot.java]: unnecessary space. Check it, please

**Review by svetamolchanova — COMMENTED** [review]
very good

File: src/main/java/by/antonsh/lesson9/Ex28/ObjectEx.java (hunk 1 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson9/Ex28/ObjectEx.java]: Ok, i delete

File: src/main/java/by/antonsh/lesson9/robot/Robot.java (hunk 2 above)
Comment (antonbl32) [reply src/main/java/by/antonsh/lesson9/robot/Robot.java]: Ok

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**